modules/bg_controller/templates/merge-buildspec-source.yml.tpl

```diff
@@ -24,11 +22,20 @@ phases:
       - printf "%s\n%s\nus-east-1\njson" | aws configure --profile ${aws_profile}
       - |
         if [ "$DEPLOYMENT_TYPE" != "AppMesh" ]; then
-          aws deploy put-lifecycle-event-hook-execution-status --deployment-id $DEPLOYMENT_ID --lifecycle-event-hook-execution-id $HOOK_EXECUTION_ID --status Succeeded --output text
-          DEPLOY_STATUS=$(aws deploy get-deployment --deployment-id $DEPLOYMENT_ID --query 'deploymentInfo.status' --output text)
-          if [ "$DEPLOY_STATUS" = "InProgress" ] || [ "$DEPLOY_STATUS" = "Ready" ]; then
-            aws deploy continue-deployment --deployment-id $DEPLOYMENT_ID --deployment-wait-type TERMINATION_WAIT
-          fi
+          DEPLOYMENT_DETAILS=$(aws dynamodb get-item --table-name MergeWaiter-${app_name}-${env_type} --key '{"APPLICATION" :{"S":"${app_name}-${env_name}"}}' --attributes-to-get '["Details"]' --query 'Item.Details.L[].M') 
```

Comment:
Put the table variables as global params for all the sections and reuse the variable instead of hardcoded strings.
I.e.
dynamo-db-params:
TableName: "..
DetailsAttrib: "..
.
.

Reply:
the table name is a variable 'MergeWaiter-${app_name}-${env_type}' app_name  and env_name are passed from the pipeline, 'Details' is static it should not change